docs/application/dotnet/guides/applications/uiapplication/application_lifecycle.md

```diff
@@ -0,0 +1,186 @@
+# Application lifecycle
+
+The Tizen .NET application model handles application lifecycle and system events. The Tizen .NET application lifecycle is handled by the [Tizen.Applications](/application/dotnet/api/TizenFX/latest/api/Tizen.Applications.html) namespace.
+
+It provides methods to manage the main event loop, the application state change events, and basic system events. Applications in the same package follow the same installation lifecycle, which is handled by the application package manager.
+Each application in the package follows its own application lifecycle. Each application in an application package can be identified by its own ID.
+
```

Comment:
As per comment provided (https://github.com/Samsung/tizen-docs/pull/1493#discussion_r715279749), can we include the description (Line 164 - line 182) currently added at the end to here?

Also, could you please follow the structure as followed in other guides for this guide, that way reader finds it uniform? Please see following guide structure as reference.
(EX: https://docs1.stg.tizen.org/staging/1493/application/dotnet/guides/applications/ui-app/)